sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -380,13 +380,15 @@ async def _read(
     async def _write(self, s):
         """Write a string out to the SSL socket fully."""
         self.writer.write(s)
+        await self.writer.drain()
```

Comment:
checking that these changes are already in sync/not needed?

Reply:
not needed in sync :)